js/StandardLibraryIntellisense.ts

```diff
@@ -27,6 +33,15 @@ class Sketch {
     AddWire (wire                : oc. TopoDS_Wire) : Sketch;
     
     LineTo  (nextPoint           : number[]       ) : Sketch;
+    HorizontalLineTo(xNextPoint: number): Sketch;
+    HorizontalLineOfLength(xLength: number): Sketch;
+    VerticalLineTo(yNextPoint: number): Sketch;
+    VerticalLineOfLength(yLength: number): Sketch;
+    AngledLineOf(angle: number, lengthOptions: {ofLength?: number, ofVerticalLength?: number, ofHorizontalLength?: number}): Sketch;
```

Comment:
Not sure if I like the API of this or not. The the idea with the object is that only one of these params is needed making it flexible because for a sloped line you might know it's length, or only how high etc it is. Of course, the user can do their own trig but I figure this is a quality of life and readability improvement.

Names should probably be changed to `ofLength`, `ofVLength` and `ofHlength` if we change the other names?

A did think about making it very abstract that you could give it the angle of your line, but it's length would be defined by the projected length of another line at different angle. So `AngledLineOf(angle, {projectedLength: someLength, projectedAngle: 90 degrees})` would effectively be the same as `ofVerticalLength` above. But I'm sure you'll agree it's too abstract, vague and don't think it would be used.